src/client/components/pinboard-measure-tile/pinboard-measure-tile.mocha.tsx

```diff
@@ -25,14 +25,16 @@ import { SortOn } from "../../../common/models/sort-on/sort-on";
 import { renderIntoDocument } from "../../utils/test-utils";
 import { PinboardMeasureTile } from "./pinboard-measure-tile";
 
-describe("PinboardMeasureTile", () => {
+describe.skip("PinboardMeasureTile", () => {
```

Comment:
is is deliberate or was it just a temporary solution that got commited?
maybe we should just remove it?

Reply:
Handled in #351